2024-07-01 08:52:23.451 [info] (Client) Pylance async client (2024.6.1) started with python extension (2024.8.1)
2024-07-01 08:52:24.690 [info] [Info  - 8:52:24 AM] (343) Pylance language server 2024.6.1 (pyright version 1.1.364, commit 0618acc5) starting
2024-07-01 08:52:24.777 [info] [Info  - 8:52:24 AM] (343) Server root directory: file:///home/ndao/.vscode-server/extensions/ms-python.vscode-pylance-2024.6.1/dist
2024-07-01 08:52:24.881 [info] [Info  - 8:52:24 AM] (343) Starting service instance "DB_Provisionning"
2024-07-01 08:52:24.920 [info] Auto-Indent enabled
2024-07-01 08:52:24.944 [info] Auto-Indent enabled
2024-07-01 08:52:24.946 [info] [Info  - 8:52:24 AM] (343) Setting pythonPath for service "DB_Provisionning": "/bin/python3"
2024-07-01 08:52:24.946 [info] [Info  - 8:52:24 AM] (343) Setting environmentName for service "DB_Provisionning": "3.8.10 (global)"
2024-07-01 08:52:25.059 [info] [Info  - 8:52:25 AM] (343) Assuming Python version 3.8.10.final.0
2024-07-01 08:52:25.421 [info] [Info  - 8:52:25 AM] (343) No source files found.
2024-07-01 08:52:25.427 [info] Auto-Indent enabled
2024-07-01 08:52:25.586 [info] [Info  - 8:52:25 AM] (343) Background analysis(1) root directory: file:///home/ndao/.vscode-server/extensions/ms-python.vscode-pylance-2024.6.1/dist
2024-07-01 08:52:25.590 [info] [Info  - 8:52:25 AM] (343) Background analysis(1) started
2024-07-01 09:01:16.209 [info] Auto-Indent enabled
2024-07-01 09:01:16.211 [info] [Info  - 9:01:16 AM] (343) Setting pythonPath for service "DB_Provisionning": "/bin/python3"
2024-07-01 09:01:16.211 [info] [Info  - 9:01:16 AM] (343) Setting environmentName for service "DB_Provisionning": "3.8.10 (global)"
2024-07-01 09:01:16.317 [info] [Info  - 9:01:16 AM] (343) Assuming Python version 3.8.10.final.0
2024-07-01 09:01:16.760 [info] [Info  - 9:01:16 AM] (343) No source files found.
2024-07-01 09:01:16.763 [info] Auto-Indent enabled
2024-07-01 09:01:16.787 [info] Auto-Indent enabled
2024-07-01 09:01:16.787 [info] [Info  - 9:01:16 AM] (343) Setting pythonPath for service "DB_Provisionning": "/bin/python3"
2024-07-01 09:01:16.787 [info] [Info  - 9:01:16 AM] (343) Setting environmentName for service "DB_Provisionning": "3.8.10 (global)"
2024-07-01 09:01:16.823 [info] [Info  - 9:01:16 AM] (343) Assuming Python version 3.8.10.final.0
2024-07-01 09:01:17.107 [info] [Info  - 9:01:17 AM] (343) No source files found.
2024-07-01 09:01:17.109 [info] Auto-Indent enabled
